File: TravelBug/TravelBug.Entities/Blog.cs
using System.Collections.Generic;
using TravelBug.Entities.UserData;

namespace TravelBug.Entities
{
    public class Blog : Base
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public string Coordinates { get; set; }
        public string Location { get; set; }
        public virtual AppUser User { get; set; }
        public virtual ICollection<Image> Images { get; set; } = new List<Image>();
        public virtual ICollection<Comment> Comments { get; set; } = new List<Comment>();
    }
}

File: TravelBug/TravelBug.Web/ClientApp/src/app/models/blog.ts
import { Profile } from "./profile";
import { Photo } from "./image";
import { BlogComment } from "./comment";

export class Blog {
  id: string;
  user: Profile;
  images: Photo[];
  title = "";
  description = "";
  coordinates = "51.45395348950013,-0.9786673543780711";
  created: Date;
  date = "";
  comments: BlogComment[];
}
